hero-character-card-front/script.js

```diff
@@ -67,7 +67,8 @@ $('#inputImageFile').on('input', function (e) {
 let loadedUserImages = {
```

Comment:
Not sure how much scope this adds, but could we define this as a map, or at least write variables for the keys like `const BACKGROUND_ART = "backgroundArt";`? I'm a little concerned at how many `loadedUserImages["stringThatMightHaveATypoWeCouldMiss"]` statements are showing up above.

Reply:
Very reasonable fear! I'm working on this in a separate branch now:  https://github.com/sjzhu/unitys-workshop/compare/sjzhu--add-cc-json...sjzhu:unitys-workshop:sjzhu--make-globals-for-image-purposes?expand=1

Reply:
Almost a year later, I return to this PR and have made a lot of global constants 😂